api/v1/api.go

```diff
@@ -280,16 +309,14 @@ func (papi *PapiConnection) CreateIsiVolume(name string) (resp *getIsiVolumesRes
 	return resp, err
 }
 
-// Query the attributes of a docker volume on the cluster
-// Note: A Docker Volume is just a directory on the cluster
+// Query the attributes of a volume on the cluster
```

Comment:
//GetIsiVolume queries the .. is the proper format here to adhere to go docs.  All exported functions should have comments that start with the function name.